Cirrious.FluentLayout/FluentLayout.cs

```diff
@@ -153,7 +154,7 @@ private void ThrowIfSecondItemAlreadySet()
 
         public IEnumerable<NSLayoutConstraint> ToLayoutConstraints()
         {
-            var constraint = NSLayoutConstraint.Create(
+            Constraint = NSLayoutConstraint.Create(
```

Comment:
You had mentioned that you were always calling `ToLayoutConstraints()` anyway, which is why it seemed reasonable to just use the result of that as the workaround for now. I just don't like the idea of having a property that is mysteriously not initialized until after that method is called, since it makes for a confusing contract for consumers of the class.

The ultimate solution here, in my opinion, is to rework the class entirely to make it initialize and expose anything it needs to automatically to avoid this order of operations requirement. This would end up being a breaking change so would require a major version bump, which is fine of course, but as part of that we should also look at other opportunities to clean up the API as well while we're at it.